src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -197,39 +199,38 @@ private Value GetSpanInfoValue(FileLinePositionSpan lineSpan)
             return Value.Create(builder.ToImmutableAndFree(), this);
         }
 
-        private Value GetPropertiesValue(Issue issue)
+        private Value GetPropertiesValue(Result result)
         {
             var builder = ArrayBuilder<KeyValuePair<string, Value>>.GetInstance();
 
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Severity, issue.Severity.ToString()));
-            if (issue.Severity == DiagnosticSeverity.Warning)
+            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Severity, result.Severity.ToString()));
+            if (result.Severity == DiagnosticSeverity.Warning)
             {
-                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.WarningLevel, issue.WarningLevel.ToString()));
+                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.WarningLevel, result.WarningLevel.ToString()));
             }
 
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.DefaultSeverity, issue.DefaultSeverity.ToString()));
+            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.DefaultSeverity, result.DefaultSeverity.ToString()));
 
-            if (!string.IsNullOrEmpty(issue.Title))
+            if (!string.IsNullOrEmpty(result.Title))
             {
-                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Title, issue.Title));
+                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Title, result.Title));
             }
 
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Category, issue.Category));
+            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.Category, result.Category));
 
-            if (!string.IsNullOrEmpty(issue.HelpLink))
+            if (!string.IsNullOrEmpty(result.HelpLink))
             {
-                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.HelpLink, issue.HelpLink));
+                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.HelpLink, result.HelpLink));
             }
 
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.IsEnabledByDefault, issue.IsEnabledByDefault.ToString()));
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.IsSuppressedInSource, issue.IsSuppressedInSource.ToString()));
+            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.IsEnabledByDefault, result.IsEnabledByDefault.ToString()));
 
-            if (issue.CustomTags.Count > 0)
+            if (result.CustomTags.Count > 0)
             {
-                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.CustomTags, issue.CustomTags.WhereNotNull().Join(";")));
+                builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.CustomTags, result.CustomTags.WhereNotNull().Join(";")));
```

Comment:
If you are asking specifically whether Roslyn should choose to define a property `customTags` whose value is a semicolon-delimited list, or whether it should instead define a set of elementary properties whose values are Boolean or enumerated, I'd choose the latter, and @michaelcfanning suggests.

On the larger issue of whether tool vendors should be _allowed_ to define such "composite" properties, the spec is silent, and I think it should remain so.